api/requirements.txt

```
@@ -34,7 +34,7 @@ redis[hiredis]~=5.0.3
 openpyxl==3.1.2
 chardet~=5.1.0
 python-docx~=1.1.0
-pypdfium2==4.16.0
+pypdfium2==4.17.0
```

How about using the `~=` version operator instead?

```suggestion
pypdfium2~=4.17.0
```